### PR TITLE
Improve certbot and nginx roles to handle existing default files

### PR DESCRIPTION
**Pull request by TheLastCicada**
...... and setup Nginx for CADT

### CODE REVIEW

File: cadt-ui/templates/cadt.local.conf.j2 [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    ssl_certificate /etc/letsencrypt/live/{{ certbot_domains | first }}/fullchain.pem;
+    ssl_certificate_key /etc/letsencrypt/live/{{ certbot_domains | first }}/privkey.pem;
+
+    ssl_protocols             TLSv1.2 TLSv1.3;
+    ssl_prefer_server_ciphers on;
+    ssl_ciphers               ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384;
+    ssl_buffer_size           8k;
+    ssl_session_cache         shared:SSL:50m;
+    ssl_session_timeout       1d;
+    ssl_session_tickets       off;
```
Comment (cmmarslender) [inline cadt-ui/templates/cadt.local.conf.j2:33]: I think you can get these (or at least most of them) from this https://github.com/Chia-Network/ansible-roles/blob/main/certbot-cert/tasks/main.yml#L77:L78

Which is the latest best practice recommendations from lets encrypt

File: cadt-ui/templates/cadt.local.conf.j2 [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+    resolver                  8.8.8.8;
```
Comment (cmmarslender) [inline cadt-ui/templates/cadt.local.conf.j2:35]: I'm inclined to say use 1.1.1.1 for this, since our dns is on cloudflare, but not a blocker

**Review by cmmarslender — CHANGES_REQUESTED** [review]

File: cadt-ui/templates/cadt.local.conf.j2 (hunk 1 above)
Comment (TheLastCicada) [reply cadt-ui/templates/cadt.local.conf.j2]: Changed it - using the include now

File: cadt-ui/templates/cadt.local.conf.j2 (hunk 2 above)
Comment (TheLastCicada) [reply cadt-ui/templates/cadt.local.conf.j2]: updated to 1.1.1.1

**Review by cmmarslender — APPROVED** [review]

**Pull request merged**